src/gatling/resources/conf/application.conf

```diff
@@ -2,6 +2,10 @@ baseUrl = "http://localhost:8800/drafts"
 baseUrl = ${?DRAFT_STORE_BASE_URL}
 
 auth {
+  idam {
+    url = "http://localhost:8080"
+    url = ${?IDAM_URL}
+  }
   s2s {
     leaseUrl = "http://localhost:8081/testing-support/lease"
     leaseUrl = ${?S2S_LEASE_URL}
```

Comment:
I will make config field names and env var names consisted in next pull request.